addon/modifiers/did-update.js

```diff
@@ -58,7 +59,7 @@ import { setModifierManager, capabilities } from '@ember/modifier';
 */
 export default setModifierManager(
   () => ({
-    capabilities: capabilities('3.13', { disableAutoTracking: true }),
+    capabilities: capabilities(gte('3.22.0') ? '3.22' : '3.13', { disableAutoTracking: true }),
```

Comment:
Need to chat with @pzuraq about this one (did-insert and will-destroy seem fine). Specifically RE: `disableAutoTracking`. If we move to using 3.22 version of modifier managers we would not get rerendered when our own args change (kinda the point of `did-update` IMHO).

I don't know what the intersection of using the args proxy **and** using `disableAutoTracking` will do...

Reply:
My guess is that this is why the tests are failing in 3.22:

```
not ok 5 Chrome 62.0 - [61 ms] - Integration | Modifier | did-update: it basically works
    ---
        actual: >
            null
        stack: >
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:167:21)
                at processModule (http://localhost:7357/assets/test-support.js:3963:17)
                at module$1 (http://localhost:7357/assets/test-support.js:3988:4)
                at Module.callback (http://localhost:7357/assets/tests.js:165:21)
                at Module.exports (http://localhost:7357/assets/vendor.js:111:32)
                at requireModule (http://localhost:7357/assets/vendor.js:32:18)
                at TestLoader.<anonymous> (http://localhost:7357/assets/test-support.js:14870:11)
                at TestLoader.require (http://localhost:7357/assets/test-support.js:14860:27)
        message: >
            Expected 4 assertions, but 0 were run
        negative: >
            false
        browser log: |
```

Reply:
I think I figured it out?